controllers/TweetsController.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
       })
       .then(tweet => {
         createdTweet = tweet;
-        foundUser.tweets.push(tweet);
+        foundUser.tweets.push(tweet.toJSON());
```

Comment:
Since you have `tweets` set as a simple array and not a collection of tweet models, you have to JSONify the model to make it a basic object before adding it.